crates/autopilot/src/limit_orders/quoter.rs

```diff
@@ -47,7 +47,7 @@ impl LimitOrderQuoter {
         loop {
             let sleep = match self.update().await {
                 // Prevent busy looping on the database if there is no work to be done.
-                Ok(true) => Duration::from_secs_f32(10.),
+                Ok(true) => Duration::from_secs_f32(1.),
```

Comment:
This makes e2e tests faster and during normal operation 1 second sleep is fine too so it doesn't need to be configurable.